src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -926,14 +942,15 @@ private void parseProperties(final Node root) throws GameParseException {
       data.getProperties()
           .addEditableProperty(new NumberProperty(Constants.AI_BONUS_DEFENSE, null, data.getDiceSides(), 0, 0));
     }
+    properties.set(Constants.MAP_NAME, mapCodeName);    // Overwrite map name with containing folder
```

Comment:
I'm a bit confused by the naming:

`mapCodeName` +  `// Overwrite map name with containing folder`

Does that mean that `mapCodeName` == `mapFolder`? or `mapContainingFolder`?